### PR TITLE
ALLOW_AUTO_BATCHING:NO does not work for AUTO CTPUT

### PR DESCRIPTION
**Pull request by wgzintel**
### Details:
 - *setting ALLOW_AUTO_BATCHING:NO does not work for AUTO CTPUT*

### Tickets:
 - *ticket-92075*

### CODE REVIEW

File: src/plugins/auto/auto_schedule.cpp [hunk 1]
```diff
@@ -166,6 +166,10 @@ void AutoSchedule::init(const ScheduleContext::Ptr& sContext) {
         _loadContext[ACTUALDEVICE].deviceInfo.config[CONFIG_KEY(PERF_COUNT)] =
             _autoSContext->_needPerfCounters ? InferenceEngine::PluginConfigParams::YES
                                              : InferenceEngine::PluginConfigParams::NO;
+        ov::AnyMap option;
```
Comment (peterchen-intel) [inline src/plugins/auto/auto_schedule.cpp:169]: 1. If this option variable is necessary?  2. Add test cases to cover both ALLOW_AUTO_BATCH and DEVICE_BIND_BUFFER

File: src/plugins/auto/auto_schedule.cpp [hunk 2]
```diff
@@ -166,6 +166,9 @@ void AutoSchedule::init(const ScheduleContext::Ptr& sContext) {
         _loadContext[ACTUALDEVICE].deviceInfo.config[CONFIG_KEY(PERF_COUNT)] =
             _autoSContext->_needPerfCounters ? InferenceEngine::PluginConfigParams::YES
                                              : InferenceEngine::PluginConfigParams::NO;
+        _loadContext[ACTUALDEVICE].deviceInfo.config[CONFIG_KEY(ALLOW_AUTO_BATCHING)] =
+            _autoSContext->_batchingDisabled ? InferenceEngine::PluginConfigParams::NO
+                                             : InferenceEngine::PluginConfigParams::YES;
```
Comment (peterchen-intel) [inline src/plugins/auto/auto_schedule.cpp:171]: Add test cases to cover both ALLOW_AUTO_BATCH and DEVICE_BIND_BUFFER

Comment (songbell) [reply src/plugins/auto/auto_schedule.cpp:171]: is this PR still valid? allow_auto_batch will be a core config from river's implementation https://github.com/openvinotoolkit/openvino/pull/13959
@peterchen-intel @wangleis

Comment (riverlijunjie) [reply src/plugins/auto/auto_schedule.cpp:171]: PR 13959 will set `ALLOW_AUTO_BATCHING`  property to auto plugin by default.

**Comment by akladiev** [comment]
This PR will be closed in 2 weeks in case of no activity.

**Pull request closed without merging**